assets/src/application/ui/input/InlineEditInput/InlineEditText.tsx

```diff
@@ -1,10 +1,48 @@
 import React from 'react';
+import Dotdotdot from 'react-dotdotdot';
+
+import { Textfit } from 'react-textfit';
```

Comment:
I've added these 2 deps because I wanted to focus first on the result and later replace with our custom-built solution or even better - clone the already existing package